journal/modernized/13-chapter-10.adoc

```diff
@@ -504,7 +504,7 @@ When it was fixed for us to remove here,
 I had the offer of some soldiers to guard the premises, but declined it;
 as I consider that watchmen selected from our workmen are quite sufficient.
 It is necessary to keep a good look out, as the whole of the premises are built of wood,
-and are consequently very inflammable; and as the property belongs to government,
+and are consequently very flammable; and as the property belongs to government,
```

Comment:
I know that these words mean the same thing, but I'm not sure that everyone does so I was thinking it would be better to have the one everyone does know.